src/App.js

```diff
@@ -1,25 +1,30 @@
-import './App.css';
-import { Container} from "react-bootstrap";
-import {BrowserRouter, Route} from "react-router-dom";
-import NewCardForm from "./components/NewCardForm";
-import DiaryHome from "./Pages/DiaryHome";
-import LoginPage from "./Pages/LoginPage";
-import Header from "./components/Header";
+import React from 'react'
+import { Container } from 'react-bootstrap'
+import { BrowserRouter, Route } from 'react-router-dom'
+import DiaryHome from './Pages/DiaryHome'
+import LoginPage from './Pages/LoginPage'
+import Header from './components/Header'
+import { useSelector } from 'react-redux'
 
-function App() {
-    return (
-        <BrowserRouter>
-            <Container fluid className={'vh-100 main'}>
-                <Header/>
-                <Route exact path={'/'} component>
-                    <DiaryHome/>
+function App () {
+  const login = useSelector(state => state.login)
+  return (
+      <BrowserRouter>
+        <Header/>
+        <Container fluid className={'vh-100 main no-scroll'}>
+          {
```

Comment:
Find out better way to handle routing. This is fine for this project. But this is not the perfect scenario.